inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
```

Comment:
**Suggestion:** Ensure that 'kw_only=True' is necessary for the dataclass, as it requires keyword arguments for initialization which might not be desired in all use cases. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
@dataclass(frozen=True)
class ApplicationAddSourceParameters:
```